pkg/apis/autoscaling/annotation_validation.go

```diff
@@ -227,6 +230,8 @@ func validateMetric(annotations map[string]string) *apis.FieldError {
 			switch metric {
 			case CPU:
 				return nil
+			case Memory:
```

Comment:
nit: This can be put in one line with the case statement above, like `case CPU, Memory`